Todo TN1.md

```diff
@@ -0,0 +1,48 @@
+Todo TN1
+---------
+
+#Obscuro:
+
+- evm /statedb - finalise & test properly (make sure it works through reorgs)
+
+- k/v on top of edgelessdb
+
+- add support for multiple supported erc20s (mapping L1/L2 address + withdrawals)
+
+- Implement the obscuro cryptography:
```

Comment:
This is T8

Reply:
yeah, but with more granularity

Reply:
First iteration, same key every block. Next iteration would have better key derivation, one key per block